src/amo/components/SurveyNotice/index.js

```diff
@@ -39,6 +39,9 @@ type InternalProps = {|
   wasDismissed: boolean,
 |};
 
+// Important: we don't use this component right now. It was removed from
+// `AppBanner` in https://github.com/mozilla/addons-frontend/pull/10129
+// because it wasn't needed.
```

Comment:
If we're not using the component, delete it?  Unused code starts to smell.

Reply:
I agree but @bobsilverberg wanted to keep it..

Reply:
I actually never said that exactly. I was just debating the merits of keeping it vs deleting it.

Reply:
@bobsilverberg are you ok to delete the component then?

Reply:
Yes